Homework_1/Task-2.py

```diff
@@ -0,0 +1,19 @@
+# *Задача 2:*
+# Найдите сумму цифр трехзначного числа. 
+
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+c = input("Введите трехзначное число: ")
+
+с1 = int(c)//100
+с2 = int(c)//10 - с1*10
+с3 = int(c) - с1*100 - с2*10
```

Comment:
Нет необходимости, Александр, трижды использовать функцию `int`.
```suggestion
c = int(input("Введите трехзначное число: "))

с1 =  c // 100
с2 = c // 10 - с1 * 10
с3 = c - с1 * 100 - с2 * 10
```

Reply:
Спасибо, учту!